ext/dcos-installer/dcos_installer/action_lib.py

```diff
@@ -377,6 +377,10 @@ def _add_prereqs_script(chain):
 sudo mkdir -p /etc/systemd/system/docker.service.d
 sudo tee /etc/systemd/system/docker.service.d/override.conf <<- EOF
 [Service]
+Restart=always
+StartLimitInterval=0
+RestartSec=15
+ExecStartPre=-/sbin/ip link del docker0
```

Comment:
@malnick or @mellenburg do we have integration test coverage for this code in the `deploy-vpc` tests?